baker/formatWordpressPost.tsx

```diff
@@ -476,6 +477,26 @@ export const formatWordpressPost = async (
         }
     })
 
+    const expandableParagraphs = cheerioEl('block[type="expandable-paragraph"]')
+    expandableParagraphs.each((_, eP) => {
+        const $el = cheerioEl(eP)
+        const text = $el.text() || ""
```

Comment:
By just taking `.text()` here, you strip out any inline formatting, like boldness or links.

These should be preserved.